### ISSUE TITLE
The attention mask and the pad token id were not set. 

### ISSUE THREAD

**Issue by homily707** [TabbyML/tabby#54]
I follow the quick start, then visit the http://127.0.0.1:5000/_admin/

it shows
triton | down   server,vector,dagu | live
Congrats, your server is live!
To get started with Tabby, you can either install the extensions below or use the [Editor](http://127.0.0.1:5000/_admin/Editor).

but when i go to Editor, it can't complete for my code.
I go to the log, and got this：
```
2023-04-07 09:45:20,415 DEBG 'server' stderr output:
As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

2023-04-07 09:45:20,415 DEBG 'server' stderr output:
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.
```

what should i do

**Comment by homily707** [comment]
no log about why triton is down

**Comment by wsxiaoys** [comment]
Could you provide the command you used to start docker process? @homily707

**Comment by homily707** [comment]
same as quick start
```
docker run \                                        
  -it --rm \
  -v <Absolute Path>/tabby/data:/data \
  -v <Absolute Path>/tabby/data/hf_cache:/home/app/.cache/huggingface \
  -p 5000:5000 \
  -e MODEL_NAME=TabbyML/J-350M \
  tabbyml/tabby
```

**Comment by wsxiaoys** [comment]
Thanks, could you try again with --gpus all? Please note that we have updated the README to reflect our removal of Python backend support. Currently, we only support CUDA environments for downstream applications. We apologize for any inconvenience this may cause."
